tileview/src/main/java/com/qozix/tileview/markers/MarkerLayout.java

```diff
@@ -199,6 +227,12 @@ protected boolean checkLayoutParams( ViewGroup.LayoutParams layoutParams ) {
      */
     public Float anchorY = null;
 
+    /**
+     * Flag to determine if anchorX/Y are relative or absolute values
+     */
+    public int typeAnchorX = 0;
+    public int typeAnchorY = 0;
```

Comment:
please follow the convention where non-public member variables are prefixed with `m`

Reply:
They are actually public. But I will rename them to have better names and remove the zero initialization.

Reply:
oh you're right about the `public` access, no need for the `m` prefix